hashset.py

```diff
@@ -0,0 +1,68 @@
+# Time Complexity : Add - O(1), contains() - O(1)
+# Space Complexity : O(n^2) - Worst case
+# Did this code successfully run on Leetcode : Yes
+# Any problem you faced while coding this : Yes
+
+'''
+I have used a Double hash methodolgy, but somehow for the python version, the leetcode suggested that my code was better than only 48% python solution. I don't understand why is that and what could still be lacking, I thought of using Chain doubling
```

Comment:
Don't rely on the Leetcode submission status, it'll show less even for some of the efficient solution.